### ISSUE TITLE
[Bug]: ScrollablePane  containing sticky command bar and details list with sticky header breaks while scrolling

### ISSUE THREAD

**Issue by logvinay** [microsoft/fluentui#33366]
### Package

react, fluent ui

### Package version

8.121.6

### React version

17.0.2

### Environment

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 4.24 GB / 15.50 GB
  Browsers:
    Internet Explorer: 11.0.22621.3527
```

### Current Behavior

I have CommandBar as sticky and Detaillist with header as sticky inside ScrollablePane Component, With screen size 80% in chrome and 150% scaling in display settings of window. While scrolling up or down multiple times UI breaks.

Giving following error.
![Image](https://github.com/user-attachments/assets/7b5cdc25-646d-4e82-94f2-04d5504ba679)

### Expected Behavior

Scrolling should not break the UI.

### Reproduction

.

### Steps to reproduce

1. Add command bar, details list with header as sticky inside scrollable pane
2. Change display settings to 150% scaling
3. Change browser zoom to 80% (Chrome and Edge)
4. Scroll up and down continuously

### Are you reporting an Accessibility issue?

None

### Suggested severity

High - No workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by spmonahan** [comment]
Seems related: #16654

**Comment by spmonahan** [comment]
Hi @logvinay, I've previously fixed a similar issue (#16654). For that I put together this repro case:

https://codepen.io/seanms/pen/pvzJMgK

Can you confirm whether the issue repros with that example for you? I've not been able trigger bug with that case using your steps.

If it doesn't repro with my test case, please provide a new repro case that I can check out on Codepen.
